scaleapi/api.py

```diff
@@ -145,34 +123,34 @@ def _api_request(
                 return json
         elif res.status_code == 409 and "task" in endpoint and body.get("unique_id"):
             retry_history = res.raw.retries.history
-            # Example RequestHistory tuple
-            # RequestHistory(method='POST',
-            #   url='/v1/task/imageannotation',
-            #   error=None,
-            #   status=409,
-            #   redirect_location=None)
-            if retry_history != ():
-                # See if the first retry was a 500 or 503 error
-                if retry_history[0][3] >= 500:
-                    uuid = body["unique_id"]
-                    newUrl = f"{self.base_api_url}/tasks?unique_id={uuid}"
-                    # grab task from api
-                    newRes = self._http_request(
-                        "GET", newUrl, headers=headers, auth=auth
-                    )
-                    json = newRes.json()["docs"][0]
+
+            if retry_history and retry_history[0][3] >= 500:
+                uuid = body["unique_id"]
+                new_url = f"{self.base_api_url}/tasks?unique_id={uuid}"
+        
+                # Perform a GET request to retrieve task data
+                new_res = self._http_request("GET", new_url, headers=headers, auth=auth)
+
+                if new_res.status_code == 200:
+                    new_res_data = new_res.json()
+                    if new_res_data["docs"]:
+                        json = new_res_data["docs"][0]
+                    else:
+                        self._raise_on_response(new_res)
```

Comment:
```
self._raise_on_response(new_res)
```

this function doesn't exist, the correct function is actually mis-spelled lol

Reply:
spell check, lol
reverted back to `self._raise_on_respose(res)`